LEAF_Request_Portal/admin/templates/mod_templates.tpl

```diff
@@ -1,611 +1,21 @@
 <link rel=stylesheet href="../../libs/js/codemirror/addon/merge/merge.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/codemirror/5.62.2/theme/lucario.min.css">
```

Comment:
Should this be a local file vs cdn?

Reply:
Should I add it as local?

Reply:
I didn't notice that the theme was available locally. The Library has been changed.